### PR TITLE
Portfolio: Create mobile skeleton

### PR DESCRIPTION
**Pull request by mateo951**
- Added linters configurations
    - Github Actions
    - Webhint
    - Stylelint
- Added fix for linters configurations
- Added Header section skeleton into index and css files
- Added Headline section skeleton into index and css files
- Modified README to meet requirements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,35 @@
...
+
+Built With
+- HTML, CSS, JS
+- Linters, Node.js, Git, Flexbox
+
+[Live Demo](https://mateo951.github.io/portfolio/)
+
+To get a local copy up and running follow these simple example steps.
+- On your terminal write the following command to clone the repository
+    - git clone git@github.com:mateo951/portfolio.git
```
Comment (Shaher-11) [inline README.md]: convert into inline command by adding `command` like that

File: index.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  <title>Portfolio</title>
+</head>
+<body>
+  <header class="flex-container header-items">
+    <a href="#" id="header-welcome-link">Welcome</a>
+    <img src="images/union.png" alt="nav button">
+  </header>
+  <section class="flex-container bg-img">
+    <div>
+      <p class="headline-paragraphs" id="p1">Hey there. I’m Mateo<br><span id="p3">I'm a software developer</span></p>
```
Comment (Shaher-11) [inline index.html]: replace the p tag with h1 as it's the appropriate tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+          <a href="#">
+            <img src="images/socialLinks/github.svg" alt="Github Link">
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="images/socialLinks/angel.svg" alt="Angel Link">
+          </a>
+        </li>
+      </ul>
```
Comment (Shaher-11) [inline index.html:50]: Center the icons as now the space is not equal on both sides 

#### Original design
right1

#### Your project
wrong1

File: index.html [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    </nav>
+    <a href="#" class="bttn"><img src="img_girl.jpg" id="bttn-img" alt="Go to Projects" width="10" height="10"></a>
```
Comment (Shaher-11) [inline index.html]: The arrow is not showing 
wrong2

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @mateo951,

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:smiley::smiley:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] Check the comments below:point_down:


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 reviews remaining, If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

File: index.html (hunk 3 above)
Comment (mateo951) [reply index.html:50]: Resolving this issue was a little bit tricky :exploding_head:

File: index.html (hunk 2 above)
Comment (mateo951) [reply index.html]: Done :heavy_check_mark::muscle:

File: README.md (hunk 1 above)
Comment (mateo951) [reply README.md]: `git commit -m "Add the inline command that was requested from my reviewer to my README file"` :weight_lifting:

File: index.html (hunk 4 above)
Comment (mateo951) [reply index.html]: Yeah sorry for that silly mistake :confused: I'ts already fixed :nerd_face: 
Thanks for your help @Shaher-11!!

**Review by arslanbisharat — APPROVED** [review]
#Approved 👏  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**